proseco/tests/test_catalog.py

```diff
@@ -36,7 +36,7 @@ def test_get_aca_catalog_20603():
     """Put it all together.  Regression test for selected stars.
     """
     # Force not using a bright star so there is a GUI-only (not BOT) star
-    aca = get_aca_catalog(20603, exclude_ids=[40113544], n_fid=2, n_guide=6, n_acq=7,
+    aca = get_aca_catalog(20603, exclude_ids_acq=[40113544], n_fid=2, n_guide=6, n_acq=7,
```

Comment:
Related to the discussion in #178, maybe revert this test and then fix the expected catalog to show both guide and acq exclusion of ..3544?

Reply:
OK fair enough.